ci-operator/config/openshift/lightspeed-operator/openshift-lightspeed-operator-main.yaml

```diff
@@ -110,6 +110,9 @@ tests:
 - as: security
   optional: true
```

Comment:
once you have this passing, we should remove it from "optional"

Reply:
A Jira ticket is created to track this https://issues.redhat.com/browse/OLS-751